homework_4/betsapi/internal/api/webServer.go

```diff
@@ -0,0 +1,84 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+)
+
+const betPath = "/bet/:id"
+const customerBetsPath = "/user/:id/bets"
+const statusBetsPath = "/bets"
+
+// WebServer Api server
+type WebServer struct {
+	router             *gin.Engine
+	port               int
+	readWriteTimeoutMs int
+}
+
+// NewServer returns new server instance
+func NewServer(port, readWriteTimeoutMs int, ctrl Controller) *WebServer {
```

Comment:
```suggestion
func NewServer(port int, readWriteTimeoutMs int, ctrl Controller) *WebServer {
```

Prefer explicit types in function definitions. Yes, implicit type here seems clever and shorter but having the type explicitly defined here decreases the chance to screw something up later on.